tools/db_crashtest.py

```diff
@@ -124,8 +124,9 @@ def is_direct_io_supported(dbname):
     "max_background_compactions": 1,
     "max_bytes_for_level_base": 67108864,
     "memtablerep": "skip_list",
-    "prefixpercent": 25,
-    "readpercent": 25,
+    "prefixpercent": 0,
```

Comment:
Why prefix is disabled?

Reply:
I just want to have some test to cover non-prefix case.

Reply:
I am confused. So the tests that covered prefix cases are gone now?

Reply:
This is in simple_default_params structure. So the change disable prefix test in simple crash test. We run simple, normal and atomic flush crash test, so prefix tests are still there in 2/3 of the tests.

Reply:
Can you update this sentence in the PR description? "Make simple crash test to avoid prefix size to have more coverage." If i understood it right, you meant to say: "Limit prefix tests to only normal and atomic flush crash test, to let simple crash tests have higher coverage on non-prefix workloads."